subsys/bluetooth/audio/csip_set_member.c

```diff
@@ -736,19 +774,36 @@ static void notify(struct bt_csip_set_member_svc_inst *svc_inst, struct bt_conn
 		   const struct bt_uuid *uuid, const void *data, uint16_t len)
 {
 	int err;
+	const struct bt_gatt_attr *attr;
+
+	k_mutex_lock(&svc_inst_lock, K_FOREVER);
```

Comment:
We should never use `K_FOREVER` in the system workqueue thread. If we cannot immediately get the mutex, we should retry later